jsonnet/jsonnetfile.json

```diff
@@ -8,7 +8,7 @@
           "subdir": "contrib/mixin"
         }
       },
-      "version": "main"
+      "version": "release-3.5"
```

Comment:
IMHO we should make an exception for etcd otherwise we would lose improvements on the etcd dashboards. AFAICT the [release-4.10 branch](https://github.com/openshift/cluster-monitoring-operator/blob/a9fcd1a056d08a09911f0f909ba79f43bd092103/jsonnet/jsonnetfile.json#L4-L12) uses `main` too + we have a plan to move the etcd dashboards to the cluster-etcd operator anyway.